panoramapublic/src/org/labkey/panoramapublic/view/search/panoramaWebSearch.jsp

```diff
@@ -349,25 +366,38 @@
                                     {
                                         xtype: 'textfield',
                                         fieldLabel: 'Protein Name',
+                                        name: proteinNameItemId,
                                         itemId: proteinNameItemId,
-                                        labelCls: 'labkey-form-label',
+                                        labelAlign: 'top',
                                         labelWidth: 125,
                                         listeners: {
                                             render: function (comp, eOpts) {
                                                 checkAndFillValuesFromUrl(proteinNameItemId, comp);
+                                                new Ext4.ToolTip({
+                                                    target : comp.getEl(),
+                                                    html: 'Required to search for proteins. You may use the name as specified by the FASTA file, or an annotation, such as a gene name, that has been loaded from an annotations file. You may comma separate multiple names.'
```

Comment:
Does comma separation actually work here? Wouldn't it just be searching for that exact text?

Reply:
it doesn't, these tips are copy paste texts. thanks for catching them.